operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/informer/InformerConfiguration.java

```diff
@@ -22,21 +22,25 @@
 
     private final PrimaryResourcesRetriever<R> secondaryToPrimaryResourcesIdSet;
     private final AssociatedSecondaryResourceIdentifier<P> associatedWith;
+    private final boolean filterOwnCreatesAndUpdateEvents;
```

Comment:
Mainly added it because of backwards compatibility. I cannot think any other reason TBH.

Also this could be a config on KubernetesDependentResource level. To send the notification to the informer. But also we might just remove it in general, and see if somebody asks for it.

Reply:
Yep, I'd rather we remove it for now and re-add it if needed.

Reply:
ok, will do!